__tests__/api.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require('../app.js');
+const request = require('supertest');
+
+const seed = require('../db/seeds/seed.js')
+const testData = require('../db/data/test-data')
+const connection = require('../db/connection.js')
+
+beforeEach(() => seed(testData))
+afterAll(() => connection.end());
+
+describe("GET /api/topics", () => {
+  describe("successful connection test(s)", () => {
+    test("200: expect page to respond with corresponding topics table", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({
+          body
+        }) => {
+          expect(body).toMatchObject(testData.topicData)
```

Comment:
Using this expect statement here for topics is ok but with wouldn't be a robust test for articles or comments as they will have an id once inserted into the database.  toMatchObject may also not be the most appropriate matcher here 🤔